keylime/models/verifier/auth_session.py

```
@@ -1,12 +1,8 @@
 import base64
 import hmac
-import threading
 import uuid
-from contextlib import contextmanager
 from datetime import timedelta
-from typing import Any, Dict, Iterator, Optional, Sequence
-
-from sqlalchemy.orm import Session
+from typing import Any, Dict, Optional, Sequence
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)